src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC } from 'react';
+import { Person } from '../../types';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+type PropType = {
+  person: Person;
+};
+
+export const PersonItem: FC<PropType> = ({ person }) => {
+  const { slug } = useParams();
+  const { sex, name, born, died } = person;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': slug === person.slug,
+      })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={classNames({ 'has-text-danger': sex === 'f' })}
+        >
+          {name}
+        </Link>
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      <td>
+        {person.mother ? (
+          <Link
+            className="has-text-danger"
+            to={`/people/${person.mother.slug}`}
+          >
+            {person.mother.name}
+          </Link>
+        ) : person.motherName ? (
+          person.motherName
+        ) : (
+          '-'
+        )}
```

Comment:
pls avoid nested ternary expressions, refactor this code
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**

Reply:
ok